docs_2.0/manual_EN/2.introduction/1.what-is-nebula-graph.md

```diff
@@ -0,0 +1,69 @@
+# What is Nebula Graph
+
+Nebula Graph is an open-source, distributed, easily scalable, and native graph database. It is capable of hosting graphs with billions of vertices and trillions of edges, and serve queries with millisecond-latency.
+
+## What is a graph database
+
+A graph database, such as Nebula Graph, is a storage engine that specializes in storing and retrieving vast networks of information. It efficiently stores data as vertices (nodes) and edges (relationships) and allows high-performance retrieval and querying of those structures. Properties can be added to both vertices and edges. Vertices can have zero to multiple tags, and edges are always directed and typed.
+
+[TODO: Add a png of a simple graph.]
+
+Graph databases are well suited for storing most kinds of data models abstracted from reality. In almost all fields, there are certain things connected to other things. In most other modeling approaches, the relationships between things are reduced to a single link without identity and attributes. Nebula Graph, as a typical native graph database, allows you to keep the rich relationships that originated from reality equally well-represented in the database without resorting to also modeling the relationships as entities.
```

Comment:
"Nebula Graph, as a typical native graph database, allows you to keep the rich relationships that originated from reality equally well-represented in the database without resorting to also modeling the relationships as entities." ==> Nebula Graph, as a typical native graph database, allows you to keep the rich relationships that 【are？】originated from reality equally well-represented in the database without resorting to 【also？】 modeling the relationships as entities.

Suggestion: Always avoid writing long and complex sentences.